src/utils/date.js

```diff
@@ -11,6 +11,11 @@ const dateFormatter = {
     return `${d.toLocaleDateString('en-US')} ${d.toLocaleTimeString('en-US')}`;
   },
 
+  parseTimestampFromSQLTimestamp(timestamp) {
+    const d = new Date(timestamp);
```

Comment:
Why don't you use `return parseTimestamp(timestamp / 1000);` ?

You just need to explain it in the docstring but would prevent duplicating the formatting part

Reply:
We actually receive a date string from ElasticSearch, like `2022-05-09T18:55:47Z`. This is why we could not reuse the code. I will add a comment stating the format we receive.

Reply:
Ah, now I get it. Maybe add a docstring to explain this and the type of the parameter. I was thinking also about changing the param name, `timestamp` makes me assume it's an integer. However, with the docstring I'm ok to keep the name.

Reply:
Done